internal/pkg/levee/testdata/src/example.com/tests/fields/tests.go

```diff
@@ -34,6 +34,14 @@ func TestDirectFieldAccess(c *core.Source) {
 	core.Sinkf("ID: %v", c.ID)
 }
 
+func TestInlinedDirectFieldAccess() {
+	// This pattern is unlikely to occur in real code.
+	// The intent is to get Field instructions in the SSA
+	// so that we can validate that those are handled correctly.
+	core.Sinkf("Data: %v", core.Source{}.Data) // want "a source has reached a sink"
```

Comment:
Consider including something other than a const-empty string as `Data`.  Even though I know it's not something we handle right now, part of me thinks this shouldn't produce a `Diagnostic`.